acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuNode.java

```diff
@@ -377,7 +377,6 @@ public String getPeerId() {
         validatorsFile.deleteOnExit();
         Files.writeString(validatorsFile.toPath(), validatorKeys.get());
         configFiles.put(validatorsFile, VALIDATORS_FILE_PATH);
-        configMap.put("validators-unencrypted-key-file", VALIDATORS_FILE_PATH);
```

Comment:
Ok, this really should break the ATs. Without this arg being set, we aren't running any validators so there shouldn't be any blocks produced.

Looks like our ATs are only checking for new blocks in tests that are using mock genesis.  So we get away with it. That's still quite a reasonable level of testing actually because mock generators still generate blocks in the same way, just a different way to load keys and create genesis.  We have a test that checks we do create genesis with real validators as well.

Expanding AT coverage is always good and we will do so in future but fine to remove this.